<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Berlitz</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="berlitzTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="bigNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>661</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 144pt &quot;Lucida Grande&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="historyNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>661</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="addContract">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>199</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Новый контракт</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="addClient">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>199</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить клиента</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Настройки подключения</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
